kubectl-cloudflow/docker/client.go

```diff
@@ -58,6 +60,24 @@ func GetClient(version string) (*client.Client, error) {
 	return client.NewClientWithOpts(client.WithVersion(version))
 }
 
+// GetVersionedClient returns a client with the proepr version
+func GetVersionedClient() (*client.Client, error) {
+	apiversion, apierr := exec.Command("docker", "version", "--format", "'{{.Server.APIVersion}}'").Output()
+	if apierr != nil {
+		return nil, fmt.Errorf("Could not get docker API version, is the docker daemon running? API error: %s", apierr.Error())
+	}
+
+	trimmedapiversion := strings.Trim(string(apiversion), "\t \n\r'")
+	client, error := GetClient(trimmedapiversion)
```

Comment:
Go code style requires not to use a reserved keyword name here as It collides with the builtin error interface. The convention is to use `err` or something similar.

Reply:
Btw there is a nice code style guide here: https://github.com/golang/go/wiki/CodeReviewComments.

Reply:
fixed here and in some other places too ..